core/api/graphql/mutation_rewind_run.graphql

```diff
@@ -0,0 +1,21 @@
+ mutation RewindRun($runName: String!, $entity: String, $project: String, $metricName: String!, $metricValue: Float!) {
```

Comment:
Is this generated or copied from somewhere? Can it be documented?

Reply:
copied from the python code, like all of our other queries. 
sure i can add some documentation